V/Valhalla/build_tarballs.jl

```diff
@@ -58,25 +84,28 @@ install_license ../LICENSE.md
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-# Windows is blocked until pkg-config issues are figured out (https://github.com/valhalla/valhalla/issues/3931)
-platforms = supported_platforms(; exclude=Sys.iswindows)
+platforms = supported_platforms()
 platforms = expand_cxxstring_abis(platforms)
 
 # The products that we will ensure are always built
 products = Product[
     LibraryProduct("libvalhalla", :libvalhalla),
+    ExecutableProduct("valhalla_service", :valhalla_service),
 ]
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("boost_jll"; compat="=1.76.0")
-    Dependency("GEOS_jll")
-    Dependency("LibCURL_jll")
+    Dependency("boost_jll"; compat="=1.87.0")
+    Dependency("GEOS_jll"; compat="3.13.1")
+    Dependency("LibCURL_jll"; compat="7.73,8")
     Dependency("Lz4_jll")
-    Dependency("protoc_jll")
+    Dependency("protoc_jll"; compat="105.29.3")
     HostBuildDependency("protoc_jll")
     Dependency("Zlib_jll")
+    # Until we have a new version of OpenSSL built for riscv64 we need to use the
+    # `get_addable_spec` hack.  From v3.0.16 we should be able to remove it here.
+    Dependency(get_addable_spec("OpenSSL_jll", v"3.0.15+2"); compat="3.0.15", platforms=filter(p -> !(Sys.iswindows(p) || Sys.isapple(p)), platforms))
```

Comment:
Not needed anymore. We have OpenSSL 3.0.16 now that supports all the platforms, so please define a compat against that version here instead.